View_Assist_custom_sentences/Play_Music_with_Music_Assistant/blueprint-playmusicwithmusicassistant.yaml

```diff
@@ -24,156 +24,192 @@ blueprint:
       description: The View Assist dashboard view to use for displaying music information (example /dashboard-viewassist/music)
       default: "/dashboard-viewassist/music"
 alias: View Assist - Play Music with Music Assistant
-description: ""
-trigger:
-  - platform: conversation
-    command:
+description: "Control volume, playback and play music using music assistant"
```

Comment:
Reword to not include control volume

Reply:
also removed control here as it's not in this blueprint either as was moved to the control BP.